owsi-core/owsi-core-examples/wicket-showcase/wicket-showcase-webapp/src/main/java/fr/openwide/core/showcase/web/application/portfolio/page/PortfolioMainPage.java

```diff
@@ -49,7 +51,7 @@ public PortfolioMainPage(PageParameters parameters) {
 		UserDataProvider userDataProvider = new UserDataProvider(searchTermModel, activeModel);
 		
 		// Porfolio
-		UserPortfolioPanel portfolioPanel = new UserPortfolioPanel("userPortfolio", userDataProvider, showcaseConfigurer.getPortfolioItemsPerPageUser());
+		UserPortfolioPanel portfolioPanel = new UserPortfolioPanel("userPortfolio", userDataProvider, ApplicationPropertyModel.of(ShowcaseWebappPropertyIds.PORTFOLIO_ITEMS_PER_PAGE_USER).getObject());
```

Comment:
UserPortfolioPanel from the showcase project is kind of outdated. It's based on AbstractGenericItemListPanel which is not longer used (see DataTableBuilder) and was made for a <code>int</code> parameter.